Services/Interfaces/web_interface/static/js/common/candlesticks.js

```diff
@@ -121,6 +121,8 @@ function get_watched_symbol_price_graph(element, callback=undefined, no_data_cal
 const stop_color = "#FFA500";
 const sell_color = "#F65A33";
 const buy_color = isDarkTheme() ? '#299a39' : "#009900";
+const candle_sell_color = isDarkTheme() ? "#F65A33" : "#F65A33";
+const candle_buy_color = isDarkTheme() ? '#7DF98D' : "#009900";
```

Comment:
I think we should use a color from our brand colors. We have few green colors : 6cb596, 82b084, a9d1a8, 65e7cf